gtsam/sfm/BinaryMeasurement.h

```diff
@@ -71,6 +71,14 @@ template <class T> class BinaryMeasurement : public Factor {
     this->noiseModel_->print("  noise model: ");
   }
 
+  // TODO: make this more general?
```

Comment:
In GTSAM we prefer to have const methods only; please refactor by passing the robust model in the constructor.